src/ResourceManager/Network/Commands.Network/Commands.Network.csproj

```diff
@@ -600,10 +600,18 @@
       <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
       <Name>Commands.Common.Authentication.Abstractions</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
```

Comment:
Do not add this reference.  You should only have a reference to the abstractions class.